sonar-plugin-api/src/main/java/org/sonar/api/server/debt/DebtRemediationFunction.java

```diff
@@ -63,35 +63,66 @@
      */
     CONSTANT_ISSUE(false, true);
 
-    private final boolean usesCoefficient;
-    private final boolean usesOffset;
+    private final boolean usesGapMultiplier;
+    private final boolean usesBaseEffort;
 
-    Type(boolean usesCoefficient, boolean usesOffset) {
-      this.usesCoefficient = usesCoefficient;
-      this.usesOffset = usesOffset;
+    Type(boolean usesGapMultiplier, boolean usesBaseEffort) {
+      this.usesGapMultiplier = usesGapMultiplier;
+      this.usesBaseEffort = usesBaseEffort;
     }
 
+    /**
+     * @deprecated since 5.5, replaced by {@link #usesGapMultiplier()}
+     */
+    @Deprecated
     public boolean usesCoefficient() {
-      return usesCoefficient;
+      return usesGapMultiplier();
+    }
+
+    public boolean usesGapMultiplier() {
+      return usesGapMultiplier;
     }
 
+    /**
+     * @deprecated since 5.5, replaced by {@link #usesBaseEffort()}
+     */
+    @Deprecated
     public boolean usesOffset() {
-      return usesOffset;
+      return usesBaseEffort();
     }
+
+    public boolean usesBaseEffort() {
+      return usesBaseEffort;
+    }
+
   }
 
   Type type();
 
   /**
-   * Non-null value on {@link Type#LINEAR} and {@link Type#LINEAR_OFFSET} functions, else {@code null}.
+   * @deprecated since 5.5, replaced by {@link #gapMultiplier()}
    */
+  @Deprecated
   @CheckForNull
   String coefficient();
 
   /**
-   * Non-null value on {@link Type#LINEAR_OFFSET} and {@link Type#CONSTANT_ISSUE} functions, else {@code null}.
+   * Non-null value on {@link Type#LINEAR} and {@link Type#LINEAR_OFFSET} functions, else {@code null}.
+   */
+  @CheckForNull
+  String gapMultiplier();
+
+  /**
+   * @deprecated since 5.5, replaced by {@link #baseEffort()}
    */
+  @Deprecated
   @CheckForNull
   String offset();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)